src/components/organisms/Carousel/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './style.css';
+import CarouselText from '../../molecules/CarouselText/index';
+import Image from '../../atoms/Image/index';
+import Button from '../../atoms/Button/index';
+import carousel_img_1 from '../../../images/carousel_img_1.png';
+import carousel_img_2 from '../../../images/carousel_img_2.png';
+import carousel_img_3 from '../../../images/carousel_img_3.png';
```

Comment:
get images location from structure.json

Reply:
Created file data.json for storing the data